src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -116,21 +116,11 @@ export default class Fms {
         */
         this._previousRouteSegments = [];
 
-        // TODO: Use this!
-        /**
-         * Airport the aircraft arrives to
-         *
-         * @for Fms
-         * @property arrivalAirport
-         * @type {AirportModel}
-         */
-        this.arrivalAirport = null;
-
```

Comment:
Including the airport model, up to two times, for each aircraft makes me very nervous. I would prefer to keep the fms as slim as practical. How do you plan to use them?

I worry about circular references and just unnecessary code.